pages/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import BasePage from '../pages/BasePage';
+
+class HomePage extends BasePage {
+
+  //WebElements
+  get title(){ return $('#page-title') }
+
+  get userConf() { return $('img[alt="User Avatar"]')};
```

Comment:
Prefer descriptive names whenever possible for any code object. Also, in the context of page objects, prefer naming them using terminology the user would use to name those elements. In this case, I think you are referring to the **user profile picture** (literally the user avatar) you see when in the home page. A better name would be, IMO: **userAvatar**